command/show.go

```diff
@@ -166,7 +166,7 @@ func (c *ShowCommand) Run(args []string) int {
 		// We're setting priorState to null because a saved plan file only
 		// records the base state (possibly updated by refresh), not the
 		// prior state (direct result of the previous apply).
-		localBackend.RenderPlan(plan, stateFile.State, nil, schemas, c.Ui, c.Colorize())
+		localBackend.RenderPlan(plan, stateFile.State, schemas, c.Ui, c.Colorize())
```

Comment:
I think this change has invalidated the comment above it, because you've removed the argument it's talking about. :grinning:

Reply:
Ha, I should have read the whole comment ;)